src/views/UserPanelView/UserPanelView.jsx

```diff
@@ -40,7 +40,9 @@ export function UserPanelView () {
       <AdminPanelContainer initialTab="users">
         <Box>
           <TitlePanel title="Users"/>
-          <UsersPanelTable classes={classes}/>
+          <div>
```

Comment:
If we are using mui components we should use Box or fragment <> </>, in this case you can use the box component